### ISSUE TITLE
Improve accessibility of graphs

### ISSUE THREAD

**Issue by jonahsnider** [Mechanical-Advantage/AdvantageScope#11]
In the future it would be nice if there was a way to configure what colors and styles lines should use. The current system works okay, but there are a few issues:
1. Poor contrast depending on background color (ex. yellow line on white background)
2. Poor accessibility for colorblind users - the lack of anything other than color to differentiate lines can make the app near-unusable for anyone who's colorblind

See this page for info on improving accessibility: https://accessibility.psu.edu/images/charts/

**Comment by jonahb55** [comment]
At least for the line graphs, this shouldn't be too difficult. I'll work on putting something together when I get the chance. It would also be worth looking more thoroughly at the rest of the app to see if there are other major issues (for example, I don't think that screen readers would behave well at all right now). We won't be able to fix everything, but there may be other ways to start moving in the right direction.
